examples/fabric/image_classifier/train_torch.py

```diff
@@ -15,18 +15,44 @@
 from os import path
 
 import torch
+import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
 import torchvision.transforms as T
-from models import Net
 from torch.optim.lr_scheduler import StepLR
 from torchvision.datasets import MNIST
 
-DATASETS_PATH = path.join(path.dirname(__file__), "..", "..", "Datasets")
+DATASETS_PATH = path.join(path.dirname(__file__), "../..", "..", "Datasets")
 
 
 # Credit to the PyTorch team
 # Taken from https://github.com/pytorch/examples/blob/master/mnist/main.py and slightly adapted.
+class Net(nn.Module):
```

Comment:
Can we import the version in `src/pytorch_lightning/demos/boring_classes.py`?

Reply:
I want to keep the file fully self-contained. This is a very trivial net, and we don't expect there to be major changes to it